src/main/kotlin/com/expedia/graphql/schema/generator/SchemaGenerator.kt

```diff
@@ -163,11 +164,18 @@ internal class SchemaGenerator(
                 .type(propertyType)
                 .deprecate(prop.getDeprecationReason())
 
-        return if (config.dataFetcherFactory != null && prop.isLateinit) {
+        prop.directives().forEach {
```

Comment:
Was this intentional that directives were not added to properties?

Reply:
our use case was just for functions. i believe spec allows it to be on arbitrary field so we should update it